crates/sui-graphql-rpc/src/types/object.rs

```diff
@@ -642,21 +655,35 @@ impl Object {
     }
 
     /// Query the database for a `page` of objects, optionally `filter`-ed.
+    ///
+    /// The `checkpoint_viewed_at` parameter is an Option<u64> representing the
+    /// checkpoint_sequence_number at which this page was queried for, or None if the data was
+    /// requested at the latest checkpoint. Each entity returned in the connection will inherit this
+    /// checkpoint, so that when viewing that entity's state, it will be from the reference of this
+    /// checkpoint_viewed_at parameter.
```

Comment:
nits: (not super important because this doc comment is only for us, but I find it's helpful to just stick with a consistent style everywhere, and then it doesn't matter if docs that you thought were internal actually get included in e.g. the GraphQL docs, because they will still look good and make sense).

- No need to mention the type -- you can see that in the signature below.
- `checkpoint_sequence_number` reads like a variable, one that isn't in scope at the moment.  Just "checkpoint sequence number".
- Conversely, `None` should be in an inline code block.

```suggestion
    /// `checkpoint_viewed_at` represents the checkpoint sequence number at which this page was
    /// queried for, or `None`  if the data was requested at the latest checkpoint. Each entity
    /// returned in the connection will inherit this checkpoint, so that when viewing that entity's
    /// state, it will be as if it was read at the same checkpoint.
```

Reply:
Thanks, you have such a way with words. Applied